lib/msal-common/src/cache/entities/AccountEntity.ts

```diff
@@ -53,15 +60,15 @@ export class AccountEntity {
     lastModificationApp?: string;
     cloudGraphHostName?: string;
     msGraphHost?: string;
-    idTokenClaims?: TokenClaims;
     nativeAccountId?: string;
+    tenantProfiles?: Array<TenantProfile>;
 
     /**
      * Generate Account Id key component as per the schema: <home_account_id>-<environment>
      */
     generateAccountId(): string {
         const accountId: Array<string> = [this.homeAccountId, this.environment];
-        return accountId.join(Separators.CACHE_KEY_SEPARATOR).toLowerCase();
+        return accountId.join("-").toLowerCase();
```

Comment:
Let's continue to use the constant here since it does have a value - we'll refactor the constants later for the size gains